test-utils/state-viewer/src/cli.rs

```diff
@@ -104,13 +101,35 @@ impl StateViewerSubCommand {
 
 #[derive(Clap)]
 pub struct DumpStateCmd {
+    /// Optionally, can specify at which height to dump state.
     #[clap(long)]
     height: Option<BlockHeight>,
+    /// Dumps state records and genesis config into separate files.
```

Comment:
so you can set both 'stream' and 'single file output' to true at the same time? What happens then?

Reply:
Updated. Those options are now mutually exclusive.